contracts/liquidity_hub/pool-manager/src/router/commands.rs

```diff
@@ -112,12 +114,13 @@ pub fn execute_swap_operations(
                         amount: vec![swap_result.burn_fee_asset],
                     }));
                 }
-
-                // todo this should be not a BankMsg but a fill_rewards msg
                 if !swap_result.protocol_fee_asset.amount.is_zero() {
-                    fee_messages.push(CosmosMsg::Bank(BankMsg::Send {
-                        to_address: config.whale_lair_addr.to_string(),
-                        amount: vec![swap_result.protocol_fee_asset],
+                    fee_messages.push(CosmosMsg::Wasm(WasmMsg::Execute {
```

Comment:
There's a function in the ww-std package (whale_lair.rs) that contains a function that builds this message for you. We can refactor this in the future.